docs/getting_started.md

```diff
@@ -1,316 +1,40 @@
-
-logger = logging.getLogger(__name__)
-
-
-def extract_dimensions(images: bytes) -> t.Tuple[np.int16, np.int16]:
-    """Extract the width and height of an image.
-
-    Args:
-        images: input dataframe with images_data column
-
-    Returns:
-        np.int16: width of the image
-        np.int16: height of the image
-    """
-    width, height = imagesize.get(io.BytesIO(images))
-
-    return np.int16(width), np.int16(height)
-
-
-class ImageResolutionExtractionComponent(PandasTransformComponent):
-    """Component that extracts image dimensions."""
-
-    def transform(self, dataframe: pd.DataFrame) -> pd.DataFrame:
-        """
-        Args:
-            dataframe: Dask dataframe
-        Returns:
-            dataset.
-        """
-        logger.info("Filtering dataset...")
-
-        dataframe[[("images", "width"), ("images", "height")]] = \
-            dataframe[[("images", "data")]].apply(lambda x:extract_dimensions(x.images.data), axis=1)
-        
-        return dataframe
-```
-This component is rather simple: it will take the images as input and extract the width and height of the images. It will then add these columns to the images subset and return the dataframe. We subclass the `PandasTransformComponent` where the user only needs to define the `transform` method. This method will be called with a pandas dataframe as input and should return a pandas dataframe as output. 
-
-The last thing we need for our component is a `Dockerfile` that specifies the steps needed to build the image our component needs:
-    
-```dockerfile
-FROM --platform=linux/amd64 python:3.8-slim
-
-## System dependencies
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install git -y
-
-# install requirements
-COPY requirements.txt /
-RUN pip3 install --no-cache-dir -r requirements.txt
-
-# Set the working directory to the component folder
-WORKDIR /component/src
-
-# Copy over src-files
-COPY src/ .
-
-ENTRYPOINT ["fondant", "execute", "main"]
 ```
-There is nothing special about this Dockerfile, it installs the python dependencies and copies over the source code of our component.
-
-And a `requirements.txt` file that specifies the python dependencies of our component:
-
-```
-fondant==0.2.0
-imagesize==1.4.1
+fondant --help
 ```
 
-With our component complete we can now add it to our pipeline definition (`pipeline.py`):
+## Demo
+For demonstration purposes, we provide sample pipelines in the Fondant GitHub repository. A great starting point is the pipeline that loads and filters creative commons images. To follow along with the upcoming instructions, you can clone the repository and navigate to the examples/pipelines/filter-cc-25m folder.
 
-```python
-extract_resolution = ComponentOp(
-    component_dir='components/extract_resolution', 
-)
+This pipeline loads an image dataset and reduces the dataset to png files. For more details on how you can build this pipeline from scratch, check out our [Tutorial](tuturials/tutorial_1.md). 
```

Comment:
We should get rid of this sentence: "For more details on how you can build this pipeline from scratch, check out our [Tutorial](tuturials/tutorial_1.md)", we say the same thing a few lines later (in the custom pipelines paragraph)